examples/connectivity/plot_cwt_sensor_connectivity.py

```diff
@@ -12,20 +12,20 @@
     physiological data in the presence of volume-conduction, noise and
     sample-size bias" NeuroImage, vol. 55, no. 4, pp. 1548-1565, Apr. 2011.
 """
-
 # Author: Martin Luessi <mluessi@nmr.mgh.harvard.edu>
 #
 # License: BSD (3-clause)
 
-print(__doc__)
-
 import numpy as np
+
 import mne
 from mne import io
 from mne.connectivity import spectral_connectivity, seed_target_indices
 from mne.datasets import sample
 from mne.time_frequency import AverageTFR
 
+print(__doc__)
```

Comment:
your pep8 is too old; top-level imports not at top of file (the execution of the print statement makes the imports not at the top)